R/fiveyeargwsum-data.R

```diff
@@ -40,7 +40,7 @@ parseFiveYrSupplemental <- function(data, parsedData){
   horizontalGrid <- signif(seq(from=seq_horizGrid[1], to=seq_horizGrid[2], along.with=seq_horizGrid), 1)
   
   startDate <- formatDates(data$reportMetadata$startDate, "fiveyr", "start")
-  endDate <- formatDates(data$reportMetadata$endDate, "fiveyr", "end")
+  endDate <- formatDates(data$reportMetadata$endDate, "fiveyr", "end") + hours(23) + minutes(45)
```

Comment:
Talked to @lindsaycarr about this and she thinks the hours 23 minutes 45 will still make a difference. I think we should look at the formatDates function for fiveYr a bit though.